src/core/vertexmodel.cpp

```diff
@@ -881,6 +936,15 @@ void VertexModel::setDirty( bool dirty )
   emit dirtyChanged();
 }
 
+void VertexModel::setAngleFonud( bool angleFonud )
```

Comment:
This becomes a int commented int he header. But also, keep the getter and setter functions together in the source code (ATM they are disjointed and it just makes the code harder to skim through).